resources/test/setupTest.js

```diff
@@ -0,0 +1,5 @@
+const sinon = require('../../node_modules/sinon');
+
+global.sinon = sinon;
```

Comment:
We should be able to remove all `sinon` references since you converted to use Jest mock functions